server/chat/backend/agent/tools/cloud_exec_tool.py

```
@@ -1096,6 +1095,45 @@ def execute_tailscale_command(command: str, isolated_env: dict) -> dict:
+            "HOME": _ISOLATED_HOME,
+            "USER": os.environ.get("USER", ""),
+            "FLY_API_TOKEN": api_token,
+        }
+
+        logger.info(f"Fly.io isolated environment configured (org: {org_slug})")
+        logger.info(f"TIME: setup_flyio_environment_isolated completed in {time.perf_counter() - fn_start:.2f}s")
+
+        return True, org_slug, "api_token", isolated_env
+
+    except Exception as e:
+        logger.error(f"Failed to setup Fly.io environment: {e}")
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)